### ISSUE TITLE
Precinct results in a single file (long-term)

### ISSUE THREAD

**Issue by CalebKleppner** [BrightSpots/rcv#105]
The precinct report currently gives complete results for ballots from a single precinct (all candidates, all rounds, details, etc) in a single Excel file.  This is 266 separate files for Minneapolis, and Minneapolis is a small city.  Ultimately, it would be nice to show vote totals by precinct for all candidates in a single file.  The way to do this is to list candidates down the first column and precincts across the rows.  Then each tab is a new round of the tabulation, and all you're showing in the round is vote totals (no transfers, percents, etc).  Check out the uploaded example, but don't worry about this.  It's not a v1.0 thing.

[RCV precinct report example.xlsx](https://github.com/BrightSpots/rcv/files/2258069/RCV.precinct.report.example.xlsx)

**Comment by CalebKleppner** [comment]
I recognize that you can't have multiple tabs in a .csv file.  Here are 3 ways can address this:

1. Switch to an .xlsx format for the precinct report and have one tab per round (boo! proprietary file format)
2. Switch to one .csv file per found (I can live with that, but it could be a silly number of files being output, e.g. 500 or 5,000 files)
3. Instead of putting round 2 in a separate tab or file, simply put round 2 *below* round 1.  The number of columns required would be basically the number of candidates.  The number of rows would be the the number of precincts times the number of rounds.  (this is probably the right answer, especially because the json output file could follow the same basic pattern).

**Comment by tarheel** [comment]
I don't have a strong opinion here. I agree that our current practice of generating a separate file per precinct is unwieldy.

I think a single file would be the best user experience. If we feel strongly about avoiding proprietary formats, though, is .ods an option? But most people don't know what that is...

**Comment by CalebKleppner** [comment]
I agree that a single file would be the best user experience.  Let's just use a .csv in my option 3 above:  candidate totals are columns, precincts are rows, and you first list round 1 vote totals (all candidates, all precincts), then below that you list round 2 totals, and so on.  So a tabulation run with precinct report will have the following outputs:

summary (.csv)
summary (.json)
precinct (.csv)
precinct (.json)
audit log (.log -- one or more files)

[RCV precinct report example one tab.pdf]
(https://github.com/BrightSpots/rcv/files/2783699/RCV.precinct.report.example.one.tab.pdf)

**Comment by gngilbert** [comment]
I think we can save this for a future enhancement.
